Backpack/HorizontalNavigation/Classes/BPKHorizontalNavigationItem.h

```diff
@@ -0,0 +1,59 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018-2019 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#import "BPKHorizontalNavigation.h"
+#import <Backpack/Button.h>
+#import <Foundation/Foundation.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+/**
+ * `BPKHorizontalNavigationItem` is a subclass of `UICollectionViewCell`.
```

Comment:
These docs are incorrect and should be placed immediately adjacent to the `@interface` declaration. Can we improve the doc string a bit too?

Reply:
Although arguably this is not public since it will not be exported to the users directly so maybe we should leave it undocumented, at least the top level doc comment which will hide it from Jazzy